src/modules/displayDOM.js

```diff
@@ -0,0 +1,38 @@
+import getShow from './getShow.js';
+import likeBtn from '../assets/like-btn.png';
+import getLikes from './getLikes.js';
+import movieCount from './totalMovies.js';
+import fetchData from './popUp.js';
+
+const displayDOM = async () => {
+  let output = '';
+  const main = document.querySelector('.main-wrapper');
+  const allShows = await getShow();
+  const shows = allShows.splice(150, 20);
+
+  shows.forEach((show) => {
+    output = `
+    <div class="movie-box">
+      <img src=${show.image.medium} alt="movie-img" class="movie-img">
+      <div class="like-line">
+      <span class="title">${show.name}</span>
+      <span class="dynLike"></span>
+      <span class="">Likes</span>
+      <img src=${likeBtn} alt="like -img" class="like-img">
+      </div>
+      <div id="btn-ctn">
+      <button id="${show.name}" data-index="${show.id}" class="comment comment-btn">comment </button>
+      </div>
+    </div>
```

Comment:
Hello @KDlamini , our team has an exceptional case from student success  because @josuerushanika  joined the team later. Therefore, they advised us not to work on the reservations section.   Thank you

Reply:
Note ✍🏾 
In the future, It would be a great idea to mention that in the PR description and also attach a screenshot of the email you received from student success.

Kindly reply with the screenshot of the email from Student Success for confirmation. 🙏🏾

Reply:
Hello @KDlamini'
I have attached to this comment a screenshot of our conversation with Marco Lizza that confirms that @josuerushanika will help us finish this project.

![image](https://user-images.githubusercontent.com/117144028/221352659-ebcd1478-784f-46eb-bad8-647654afb5fa.png)

I hope this is enough proof.

Thank You.

Reply:
Awesome!!! 💯 
You can ignore this comment. 👍🏾